apps/common/models/i-casework.js

```diff
@@ -7,6 +7,11 @@ const config = require('../../../config');
 
 module.exports = class CaseworkModel extends Model {
 
+  constructor(attributes, options) {
+    super(attributes, options);
+    this.options.timeout = this.options.timeout || 20000;
```

Comment:
any particular reason why you chose `20000` as the default? 
What is the default nginx timeout?

Reply:
Not really - it's a bit arbitrary.

From what I can work out the default in nginx is 60s, but I don't know that for certain.

On reflection, this should be made a config parameter. I'll do that in the morning.

Reply:
Citation: http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_read_timeout